layouts/highlights.html

```diff
@@ -12,6 +12,11 @@
             <p>Check out a number of <a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations">Open Source FHIR Implementations</a>.</p>
         </li>
         <li class="highlight-module">
+            <a href="/faq/"><span class="mega-octicon octicon-question"></span></a>
+            <h2><a href="/faq/">FHIR Developer FAQ</a></h2>
```

Comment:
Updated to be more generic: https://github.com/cerner/fhir.cerner.com/commit/1c26adb2c23ad42941ea18c686933277fa94e709#diff-dc155e29ed33ac86bdcb09416efdef38L16